File: app/src/main/java/com/artmurka/artmurkaapp/views/fragments/interfaces/IFragmentAboutGoods.java
package com.artmurka.artmurkaapp.views.fragments.interfaces;

import com.artmurka.artmurkaapp.model.pojo.itemlist.itemlist.GoodsProperties;

import java.util.ArrayList;

public interface IFragmentAboutGoods {

    void setName(String name);
    void setPhoto(ArrayList<String> urles);
    void setPrice(String price);
    void setDescription(String description);
    void getDataForRecyclerView(String category);

    void setFullDescription(String fullDescription);
    void setDataForRecyclerView(ArrayList<GoodsProperties> list);
    void setWishButton(boolean isInWish);
    void setBasketButton(boolean isInBasket);

}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IFragmentAboutGoods.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist.GoodsProperties
import com.artmurka.artmurkaapp.presenter.PresenterView

import java.util.ArrayList

interface IFragmentAboutGoods  : PresenterView {

    fun setName(name: String)
    fun setPhoto(urles: ArrayList<String>)
    fun setPrice(price: String)
    fun setDescription(description: String)
    fun getDataForRecyclerView(category: String)
    fun setArticle(art: String)

    fun setFullDescription(fullDescription: String)
    fun setDataForRecyclerView(list: ArrayList<GoodsProperties>)
    fun setWishButton(isInWish: Boolean)
    fun setBasketButton(isInBasket: Boolean)

}
